docs/examples.md

```diff
@@ -1,464 +1,103 @@
-          color: '#FF0000'
-      - action: request-changes@v1
-        args:
-          comment: |
-            New `LambdaFunction` must have `description:` field.
-
-```
-
-### Request changes on deprecated APIs
-
-For example, assume we have an old API `callElvis` we want to switch from to a new API `callGaga`, gitStream can review and trigger a change request automatically when the PR includes use of the deprecated API.
-
-This pattern allows defining best practices in `.cm` code. 
-
-![Request changes automatically](screenshots/change_use_deprectaed_api.png)
-
-```yaml+jinja
-automations:
-  catch_deprecated_components:
-    if:
-      - {{ source.diff.files | matchDiffLines(regex=r/callElvis/) | some }}
-    run:
-      - action: add-label@v1
-        args:
-          label: 'deprecated-component'
-          color: '#FF0000'
-      - action: request-changes@v1
-        args:
-          comment: |
-            you have used deprecated API, use `callingGaga` instead
-```
-
-gitStream supports iterators over arrays and dictionaries, so you can also make it more general:
-
-```yaml+jinja hl_lines="2 16 19"
-automations:
-  {% for item in deprecated %}
-  # Automation names should be unique, therefore the iteration number postfix
-  catch_deprecated_components_{{ loop.index }}:
-    if:
-      - {{ source.diff.files | matchDiffLines(regex=item.regex) | some }}
-    run:
-      - action: add-label@v1
-        args:
-          label: 'deprecated-component'
-          color: '#FF0000'
-      - action: request-changes@v1
-        args:
-          comment: |
-            `{{ item.old }}` component is deprecated, use `{{ item.new }}` instead
-  {% endfor %}
-
-# This list includes the deprecated items
-deprecated:
-  - regex: r/callElvis/
-    old: Elvis
-    new: Gaga
-  - regex: r/callOldAPI/
-    old: callOldAPI
-    new: callBetterAPI
-```
-
-<div class="result" markdown>
-  <span>
-  [:octicons-download-24: Download and add to your repo .cm directory](/downloads/request-changes-on-deprecated-apis.cm){ .md-button }
-  </span>
-</div>
-
+![Missing Tests Label](screenshots/missing-labels-example.png)
+
+![Deleted Files Label](screenshots/github-deleted-files.png)
+## Approval Automation
+Not all PRs need extensive review policies that loop in multiple experts. For minor changes to docs, tests, formatting, and other components that pose little risk, gitStream can automate the review process to keep your team focused on their work.
+
+!!! example "PR Approval Automations"
+    ```yaml+jinja
+    --8<-- "docs/downloads/approvals.cm"
+    ```
+    <div class="result" markdown>
+      <span>
+      [:octicons-download-24: Download this example as a CM file.](/downloads/approvals.cm){ .md-button }
+      </span>
+    </div>
+
+Here are examples of some of the automations in this configuration:
+![Approve Safe Changes](screenshots/approved-safe-changes.png)
+
+![Change Deprecated API](screenshots/change_use_deprectaed_api.png)
+## Review Assignment
```

Comment:
Review assignment is more popular then Approval Automation, might worth having this section before